android/java/org/chromium/chrome/browser/crypto_wallet/activities/AccountDetailActivity.java

```diff
@@ -177,4 +236,60 @@ public void onBackPressed() {
         setResult(Activity.RESULT_OK, returnIntent);
         finish();
     }
+
+    private AccountInfo getThisAccountInfo() {
+        AccountInfo accountInfo = new AccountInfo();
+        accountInfo.address = mAddress;
+        accountInfo.name = mName;
+        accountInfo.isImported = mIsImported;
+        return accountInfo;
+    }
+
+    @Override
+    public void onConnectionError(MojoException e) {
```

Comment:
let's put `.close()` for all Interfaces here to be safe. I'm not 100% confident we need it on connection error, but won't hurt in any case.

Reply:
force-pushed fix https://github.com/brave/brave-core/compare/65957616b4e77121c46f430d6a97f558c314d124..0a152d0975fdb6738b8de25793dac5a550a6bb66